<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportTemplateCloudView</class>
 <widget class="QDialog" name="ReportTemplateCloudView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>模板查看器</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="hospital_label">
         <property name="text">
          <string>医院:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="hospital_edit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="doctor_label">
         <property name="text">
          <string>医生:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="doctor_edit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="createtime_label">
         <property name="text">
          <string>创建时间:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="createtime_edit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="description_label">
       <property name="text">
        <string>描述:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="description_edit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="conclusion_label">
       <property name="text">
        <string>结论:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="conclusion_edit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="is_upload_to_cloud">
         <property name="text">
          <string>同步到云库</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancel_button">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ok_button">
         <property name="text">
          <string>导入</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
